dist/api_data.py

```diff
@@ -776,14 +776,6 @@ def __ge__(self, other):
             merge LSM chunks where possible''',
             type='boolean')
         ]),
-    Config('operation_timeout_ms', '0', r'''
```

Comment:
Instead of removing it altogether you should update the description to:
"this option is no longer supported, retained for backward compatibility".

Reply:
Fixed in [4de96ae](https://github.com/wiredtiger/wiredtiger/pull/9357/commits/4de96ae50ca00ba0bde584aabe445769f4f6ac27)